recipes/pennies/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pennies" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5044bfd6009dc4d6d14ed8d8355d0cc4b9cb4aafad105b80b0c481f8beacdc15
+  patches:
+    - 0001-fix-imports.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - multipledispatch
+    - numpy
+    - pandas
+    - pip
+    - pip
+    - python
+    - scipy
+  run:
+    - multipledispatch
+    - numpy
+    - pandas
+    - python
+    - scipy
+
+test:
+  imports:
+    - pennies
+    - pennies.calculators
+    - pennies.market
+    - pennies.models
+    - pennies.reporting
+    - pennies.trading
+
+about:
+  home: https://github.com/caseyclements/pennies
+  license: BSD
+  license_family: BSD
```

Comment:
The license file in the repo is APACHE-2.0. Can you please clarify? Also for now, you need to add the `license_file` manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

But it would be good to add a `MANIFEST.in` to the upstream repo and have it included in the source distribution:

https://docs.python.org/2/distutils/sourcedist.html#the-manifest-in-template